sds_data_manager/utils/stackbuilder.py

```diff
@@ -3,30 +3,30 @@
 from pathlib import Path
```

Comment:
Minor comment about naming. Keep uniform conventions. For example we have:
IAlirtBucketConstruct
IalirtProcessingPrimary

So keep or remove Construct part of the name for the Constructs.

Reply:
Yeah, this was a bit all over the place, and I didn't do a good job of consolidating. My preference would be to drop the `_construct` everywhere, see this comment:
> There is a lot of thrash just changing names from "abc_stack" to "abc_construct", might be advantageous to avoid the "_specific_name" in the future...

If you all don't mind the churn in this single PR I can do it now, but I was trying to make this a little more manageable to review.

Reply:
No worries to do it in another PR.